target/linux/ipq40xx/patches-4.14/998-arm-boot-add-ens620ext-dts-file.patch

```diff
@@ -0,0 +1,17 @@
+
+Signed-off-by: Steve Glennon <s.glennon@cablelabs.com>
+---
+ arch/arm/boot/dts/Makefile | 8 +
+ 1 file changed, 1 insertion(+)
+
+--- a/arch/arm/boot/dts/Makefile
++++ b/arch/arm/boot/dts/Makefile
+@@ -701,6 +701,7 @@
+ 	qcom-ipq4018-ap120c-ac.dtb \
+ 	qcom-ipq4018-ea6350v3.dtb \
+ 	qcom-ipq4018-eap1300.dtb \
++	qcom-ipq4018-ens620ext.dtb \
```

Comment:
No need for another patch, edit the existing 997 patch

Reply:
I was not sure which was this should go - thanks for the guidance and I have removed the proposed patch and added the line into 901 patch